macros/schema_tests/relationships_where.sql

```diff
@@ -0,0 +1,48 @@
+{% macro test_relationships_where(model, to, field) %}
+
+{% set column_name = kwargs.get('column_name', kwargs.get('from')) %}
+{% set from_condition = kwargs.get('from_condition', True) %}
```

Comment:
this is pretty minor, but I'd use `"true"` (a string) as the default value for `from_condition` and `to_condition` here

Reply:
Done.

Reply:
Great feedback. Thanks, Drew